src/Stryker.CLI/Stryker.CLI/StrykerCLI.cs

```diff
@@ -85,26 +89,39 @@ public void Run(string[] args)
                     thresholdLowParam,
                     thresholdBreakParam
                     );
-                return RunStryker(options);
+                RunStryker(options);
+                // Default ExitCode = 0
+                return Environment.ExitCode;
             });
 
             app.Execute(args);
         }
         
-        private int RunStryker(StrykerOptions options)
+        private void RunStryker(StrykerOptions options)
         {
             // start with the stryker header
             PrintStykerASCIIName();
 
             try
             {  
-                _stryker.RunMutationTest(options);
+               IStrykerRunResult results = _stryker.RunMutationTest(options);
+               
+               if(!results.isScoreAboveThresholdBreak()) 
+               {
+                   this.HandleBreakingThresholdScore(options, results);
+               }
             }
             catch (Exception)
             {
-                return 1;
+                Environment.ExitCode = 1;
             }
-            return 0;
+        }
+
+        private void HandleBreakingThresholdScore(StrykerOptions options, IStrykerRunResult results) {
+            _logger.LogError("Final mutation score: {0} under breaking threshold value {1}"
```

Comment:
This can be one LogError statement. Also, by placing an @ before the string, you can place enters inside the string. This makes the string more readable here.